### PR TITLE
Added condition to check topic is present or not

### PR DESCRIPTION
**Pull request by AayushSaini101**
### Which Issue(s) This PR Fixes

Resolve [Issue](https://github.com/apache/rocketmq/issues/7100)

Fixes #issue_id

### Brief Description

Added a condition to check topic is present or not in the cluster

### CODE REVIEW

**Review by joeCarf — COMMENTED** [review]
you may start an issue and describe this change

File: client/src/main/java/org/apache/rocketmq/client/impl/MQClientAPIImpl.java [hunk 1]
```diff
@@ -381,6 +381,11 @@ public void createTopic(final String addr, final String defaultTopic, final Topi
         throws RemotingException, MQBrokerException, InterruptedException, MQClientException {
         Validators.checkTopicConfig(topicConfig);
 
+        if (getTopicListFromNameServer(timeoutMillis).getTopicList().contains(topicConfig.getTopicName())) {
+
```
Comment (joeCarf) [inline client/src/main/java/org/apache/rocketmq/client/impl/MQClientAPIImpl.java:385]: redundant line

**Review by joeCarf — COMMENTED** [review]
you may change the title like `[ISSUE #xxxx]xxxxx`

File: client/src/main/java/org/apache/rocketmq/client/impl/MQClientAPIImpl.java (hunk 1 above)
Comment (AayushSaini101) [reply client/src/main/java/org/apache/rocketmq/client/impl/MQClientAPIImpl.java:385]: Thank you @joeCarf I will change.  Actually, i want to fetch all the topics present in the cluster but unable to find out. I checked this approach but this is not working, and causing a test case to fail. When i debug the test case. I found the getTopicList() method returning null response.

**Pull request closed without merging**